pyglet/shapes.py

```diff
@@ -76,10 +76,11 @@ def on_draw():
 
 import math
 
-from pyglet.gl import GL_COLOR_BUFFER_BIT, GL_SRC_ALPHA, GL_ONE_MINUS_SRC_ALPHA
-from pyglet.gl import GL_TRIANGLES, GL_LINES, GL_BLEND
-from pyglet.gl import glPushAttrib, glPopAttrib, glBlendFunc, glEnable, glDisable
-from pyglet.graphics import Group, Batch
+from pyglet.gl import (GL_BLEND, GL_COLOR_BUFFER_BIT, GL_LINES,
+                       GL_ONE_MINUS_SRC_ALPHA, GL_SRC_ALPHA, GL_TRIANGLES,
+                       glBlendFunc, glDisable, glEnable, glPopAttrib,
+                       glPushAttrib)
+from pyglet.graphics import Batch, Group
```

Comment:
I have vscode configured to automatically sort all import statements in Python files on hitting Cmd+S ... so I sorted them by accident. The following commit reverts the sorting again.